### DISCUSSION TITLE
Why meter is in a seperate package?

### DISCUSSION THREAD

**Discussion by davendu** [open-telemetry/opentelemetry-go#2995] [Q&A] [answered]
To create a tracer and start a new span, one can use:

```go
import "go.opentelemetry.io/otel"
//...
ctx, span := otel.Tracer("com.application/handler").Start(ctx, "current_call")
```

But to create a meter (with related counter), one should use:

```go
import "go.opentelemetry.io/otel/metric/global"
//...
meter, err := global.Meter("com.application/handler").SyncInt64().Counter("call_count")
panic_if_err(err)
```

There seems exists some inconsistency. Why `global.Meter` is not placed in `otel` package? Is there any deep reason to do so?

**Comment by MrAlias** [comment, marked as answer]
The metrics API is not stable yet, it is not covered by [our stability guarantees](https://github.com/open-telemetry/opentelemetry-go/blob/main/VERSIONING.md) and may still change. The `go.opentelemetry.io/otel` package is stable.

Once the metric API is stabilized an alias will be added to `go.opentelemetry.io/otel` for that function.

**Reply by davendu** [reply]
Thanks for the info!
